node/src/Transaction.ts

```diff
@@ -2389,6 +2390,30 @@ export class Transaction extends BaseTransaction<Transaction> {
     public select(index: number): Transaction {
         return this.addAndReturn(createSelect(index));
     }
+
+    /**
+     * Copies the value stored at the `source` to the `destination` key on `destinationDB`. When `replace`
+     * is true, removes the `destination` key first if it already exists, otherwise performs no action.
+     *
+     * See https://valkey.io/commands/copy/ for more details.
+     *
+     * @param source - The key to the source value.
+     * @param destination - The key where the value should be copied to.
+     * @param destinationDB - (Optional) The alternative logical database index for the destination key.
```

Comment:
```suggestion
     * @param destinationDB - (Optional) The alternative logical database index for the destination key. If not provided, the current database will be used.
```

Reply:
Do I make the same changes to GlideClient.ts?